lib/detect-testing-library-utils.ts

```diff
@@ -291,9 +295,21 @@ export function detectTestingLibraryUtils<
      * Otherwise, it means `custom-module` has been set up, so only those nodes
      * coming from Testing Library will be considered as valid.
      */
-    const isAsyncUtil: IsAsyncUtilFn = (node) => {
-      return isTestingLibraryUtil(node, (identifierNodeName) =>
-        ASYNC_UTILS.includes(identifierNodeName)
+    const isAsyncUtil: IsAsyncUtilFn = (node, validNames) => {
+      return isTestingLibraryUtil(
+        node,
+        (identifierNodeName, originalNodeName) => {
+          if (validNames && validNames.length > 0) {
+            return (
+              (validNames as string[]).includes(identifierNodeName) ||
+              (validNames as string[]).includes(originalNodeName)
+            );
+          }
+          return (
+            ASYNC_UTILS.includes(identifierNodeName) ||
+            ASYNC_UTILS.includes(originalNodeName)
+          );
```

Comment:
What about providing the async utils as default?
Doing this, removes the `if` statement inside `isAsyncUtil`.

```
type IsAsyncUtilFn = (
  node: TSESTree.Identifier,
  validNames: VALID_ASYNC_UTILS[]  = ASYNC_UTILS
) => boolean;
```

Reply:
Aaah of course! This is the kind of sanity check I needed 😅

Reply:
Much better now, thanks!